Formula/g/grep.rb

```diff
@@ -53,8 +53,8 @@ def install
     if OS.mac?
       %w[grep egrep fgrep].each do |prog|
         (libexec/"gnubin").install_symlink bin/"g#{prog}" => prog
-        (libexec/"gnuman/man1").install_symlink man1/"g#{prog}.1" => "#{prog}.1"
       end
+      (libexec/"gnuman/man1").install_symlink man1/"ggrep.1" => "grep.1"
```

Comment:
Let's just do this so we don't have to manually keep track of what lives in the `bin` and `man1` directories:
```suggestion
      (libexec/"gnubin").install_symlink bin.children.to_h do |file|
        [file, file.basename.to_s.delete_prefix("g")]
      end
      (libexec/"gnuman/man1").install_symlink man1.children.to_h do |file|
        [file, file.basename.to_s.delete_prefix("g")]
      end
```

Reply:
Thanks, this is a really good suggestion.

Reply:
I have not been able to make your syntax with `.to_h` working, so I ended up with

```ruby
      bin.children.each do |file|
        (libexec/"gnubin").install_symlink file => file.basename.to_s.delete_prefix("g")
      end
      man1.children.each do |file|
        (libexec/"gnuman/man1").install_symlink file => file.basename.to_s.delete_prefix("g")
      end
```